<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>691</width>
      <height>481</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tabAjouter">
     <attribute name="title">
      <string>Ajouter Etudiant</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>60</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>ID</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>150</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>240</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Prenom</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_id">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>60</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_nom">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>150</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_prenom">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>240</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonAjouter">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>290</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabConsulter">
     <attribute name="title">
      <string>Consulter Liste</string>
     </attribute>
     <widget class="QTableView" name="tableAfficahge">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>60</y>
        <width>531</width>
        <height>361</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabSupprimer">
     <attribute name="title">
      <string>Eliminer Etudiant</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_supprimer">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>180</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_supprimer">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>60</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
